src/Search/Console.php

```diff
@@ -210,7 +210,7 @@ public function __construct(Application $console, Kernel $app)
         // Some annotations
         Register::registerLoader();
 
-        $this->console->getDefinition()->addOption(new InputOption('--env', '-e', InputOption::VALUE_REQUIRED, 'The Environment name.', 'dev'));
+        $this->console->getDefinition()->addOption(new InputOption('--env', '-e', InputOption::VALUE_OPTIONAL, 'The Environment name. Deprecated and not used', 'dev'));
```

Comment:
Guess setting it causes a problem?

Reply:
Could not cause a problem, there is is some wrangling of `--env` in `bin/console`, so put this as a precaution to remove it down the line.